AdobeStockImageAdminUi/Test/Mftf/Test/AdminAdobeStockLocalizationTest.xml

```diff
@@ -43,12 +43,11 @@
         </actionGroup>
         <actionGroup ref="AdminAdobeStockExpandImagePreviewActionGroup" stepKey="expandImagePreview"/>
         <wait time="5" stepKey="waitBeforeExpandSectionLoadFull"/>
-        <!-- The following assertion/requirement on the specific keyword we expect may not be stable as we have no guarantee on the exact keyword to use as the image could be random -->
-        <actionGroup ref="AssertAdminAdobeStockImageKeywordActionGroup" stepKey="assertKeywordLocalized">
-            <argument name="keyword" value="машина"/>
+        <actionGroup ref="AssertAdminAdobeStockKeywordLocalizedActionGroup" stepKey="assertKeywordLocalized">
+             <argument name="stringToBeVerified" value="{{AdminAdobeStockImagePreviewSection.firstKeyword}}"/>
         </actionGroup>
-        <actionGroup ref="AssertAdminAdobeStockImagePreviewAttributeVisibleActionGroup" stepKey="assertCategoryLocalized">
-            <argument name="attributeName" value="Индустрия"/>
+        <actionGroup ref="AssertAdminAdobeStockKeywordLocalizedActionGroup" stepKey="assertCategoryLocalized">
+             <argument name="stringToBeVerified" value="{{imagePreviewCategoryText}}"/>
```

Comment:
Wouldn't that be {{AdminAdobeStockImagePreviewSection. imagePreviewCategoryText}} here instead?

Reply:
yep, and we need to grab the value first

Reply:
thanks :+1: